kura/org.eclipse.kura.linux.watchdog/META-INF/MANIFEST.MF

```diff
@@ -8,7 +8,10 @@ Bundle-RequiredExecutionEnvironment: JavaSE-1.6
 Service-Component: OSGI-INF/*.xml
 Bundle-ClassPath: .
 Bundle-ActivationPolicy: lazy
-Import-Package: org.eclipse.kura.configuration;version="[1.1,1.2)",
- org.eclipse.kura.watchdog;version="[1.0,1.1)",
+Import-Package: org.apache.commons.exec;version="1.3.0",
+ org.eclipse.kura;version="[1.2,2.0)",
+ org.eclipse.kura.configuration;version="[1.1,2.0)",
+ org.eclipse.kura.watchdog;version="[1.0,2.0)",
```

Comment:
``org.eclipse.kura.watchdog;version="[1.0,1.1)"`` since this bundle **provides** the implementation